### ISSUE TITLE
Crash when opening Notes for Spirits

### ISSUE THREAD

**Issue by Nedorus** [chummer5a/chummer5a#2982]
### Environment
Chummer Version: 5.202.31
Environment: (Operating system here. If it persists over multiple OS's list them all)
Runtime: (If you don't know this, it is probably .NET 4.5.1)

### Expected behaviour
On Mouse Over of the Notes icon for Spirits the Notes Popup should be shown if there is an existing note for the spirit

### Actual behaviour
When moving the mouse over the notes icon of a spirit (with existing note) chummer crashes.

### Steps to reproduce the behaviour

1. Open magician character with bound spirit with a note attached to the spirit.
2. Mouse over the notes icon

### Link to affected character file
(Enter text here)

**Comment by DelnarErsike** [comment]
Cannot reproduce this crash in the latest dev build. Can you double-check that the issue is still present in Nightly v5.202.35 and/or upload a savefile of a character for which the program crashes?

**Comment by Nedorus** [comment]
Sorry for responding late. You are always so fast in fixing stuff and replying and I try to answer quickly as well... may bad for being late.
I was able to reproduce the error with a fresh chummer file.
Here's how to reproduce:
1. create new character
2. make it a mage, increase magic attribute, add summoning skill
3. choose tradition (e.g. wuxing)
4. add a spirit and add the following string as a note to the spirit (mind the trailing "<")
`01234567890123456789012345678901234567890123456789012345678901234567890123456<`
5. Close the notes editor and mouse over the note (all should be fine)
6. Open the notes editor again and add one more digit to make the note like:
`012345678901234567890123456789012345678901234567890123456789012345678901234567<`
7. Close the notes editor and mouse over the note => crash

If you enter the longer string right away it will crash on the first try. I just did the above to show that there seems to be string length limit. If there is no "`<`" then the string can be longer... much longer!

I've created and attached a sample of a crashing file:
[Test Mage with bound Spritit and Note to Sprit.zip](https://github.com/chummer5a/chummer5a/files/2386539/Test.Mage.with.bound.Spritit.and.Note.to.Sprit.zip)
